src/platform/tls_schannel.c

```diff
@@ -2786,6 +2786,23 @@ CxPlatTlsParamGet(
             break;
         }
 
+        case QUIC_PARAM_TLS_SCHANNEL_SECURITY_CONTEXT_TOKEN:
+            if (*BufferLength < sizeof(void*)) {
+                *BufferLength = sizeof(void*);
+                Status = QUIC_STATUS_BUFFER_TOO_SMALL;
+            }
+
+            if (Buffer == NULL) {
+                Status = QUIC_STATUS_INVALID_PARAMETER;
+            }
+
+            Status =
+                SecStatusToQuicStatus(
+                QuerySecurityContextToken(
+                    &TlsContext->SchannelContext,
+                    &Buffer));
```

Comment:
`&Buffer` here should just be `Buffer`, no dereference. QuerySecurityContextToken returns a HANDLE so the current behavior doesn't crash, but the token is leaked.

Reply:
Fixed!